pyfr/solvers/aceuler/kernels/flux.mako

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<%pyfr:macro name='inviscid_flux' params='s, f'>
+    fpdtype_t v[${ndims}];
+
+    // Velocities in the first indices of the conservative variable array
+% for i in range(ndims):
+    v[${i}] = s[${i + 1}];
```

Comment:
`pyfr.array` can be used here as something like

`fpdtype_t v[] = ${pyfr.array('s[{i}]', i=(1, ndims + 1))}`;